source/index.html.md.erb

```diff
@@ -70,41 +70,43 @@ infrastructure.
+- [Cloud Platform Support](documentation/other-topics/cloud-platform-support.html)
+- [Zero Downtime Deployments](documentation/other-topics/zero-downtime-deployment.html)
+- [Using a custom domain](documentation/other-topics/custom-domain-cert.html)
+- [Creating alerts for RDS](documentation/other-topics/alert-rds.html)
+- [Accessing your RDS database](documentation/other-topics/rds-external-access.html)
+- [Viewing RDS Database Metrics](documentation/other-topics/rds-metrics-prometheus.html)
+- [RDS Snapshots](documentation/other-topics/rds-snapshots.html)
+- [Setup Postgres container](documentation/other-topics/Setup-postgres-container.html)
+- [Time Zone Cronjobs](documentation/other-topics/TZ-Cronjobs.html)
+- [How to decommission unused template deploy services](documentation/other-topics/decommission-unused-template-deploy-services.html)
+- [Git-Crypt](documentation/other-topics/git-crypt-setup.html)
+- [Kubernetes Cronjobs](documentation/other-topics/Cronjobs.html)
+- [Kubernetes jobs](documentation/other-topics/k8s-jobs.html)
+- [IP Filtering](documentation/other-topics/ip-filtering.html)
+- [Applying a Maintenance Page](documentation/other-topics/maintenance-page.html)
+- [Kubectl quick reference](documentation/other-topics/quick-reference.html)
+- [StatefulSets (Pods with Persistent Volumes)](documentation/other-topics/statefulsets.html)
+- [Storage Classes](documentation/other-topics/storage-classes.html)
+- [Troubleshooting guide](documentation/other-topics/troubleshooting.html)
+- [Long-running environments operations](documentation/other-topics/long-running-env-operations.html)
+- [Upgrading an RDS instance](documentation/other-topics/upgrade-rds-db.html)
+- [Migrating an RDS instance](documentation/other-topics/aws-rds-migration.html)
+- [SSL connections with RDS](documentation/other-topics/rds-ssl.html)
+- [Secrets overview](documentation/other-topics/secrets.html)
+- [ModSecurity - Web Application Firewall](documentation/other-topics/modsecurity.html)
+- [Network Policies](documentation/other-topics/network-policy.html)
+- [Persistent-Volume encryption and snapshot](documentation/other-topics/persistent-volume-encryption-snapshot.html)
+- [Migrating an S3 bucket](documentation/other-topics/s3-bucket-migration.html)
+- [Access cross AWS resources](documentation/other-topics/access-cross-aws-resources.html)
+- [Kibana Circleci Monitoring Dashboards](documentation/other-topics/kibana-circleci-monitoring-dashboards.html)
+- [Namespace/Container Resource Limits](documentation/concepts/namespace-limits.html)
+- [Checking the GitHub groups in your ID token](documentation/other-topics/check-id-token.html)
+- [Cloud Platform Disaster Recovery](documentation/other-topics/cloud-platform-disaster-recovery.html)
+- [Apply Pipeline](documentation/other-topics/apply-pipeline.html)
+- [Security testing and ITHC](documentation/other-topics/security-testing-and-ithc.html)
```

Comment:
This is the only addition. The rest have their indentation removed, because it's unconventional to have a space, the other blocks don't have it, and my markdown linter complains.